copy/single.go

```diff
@@ -40,6 +40,12 @@ type imageCopier struct {
 	requireCompressionFormatMatch bool
 }
 
+type copySingleImageOptions struct {
+	requireCompressionFormatMatch bool
+	compressionFormat             *compressiontypes.Algorithm // Compression algorithm to use, if the user explicitly requested one, or nil.
```

Comment:
```suggestion
	compressionFormat             *compressiontypes.Algorithm // Compression algorithm to use, if the caller explicitly requested one, or nil.
```
for future reference